docs/operations.rst

```diff
@@ -209,4 +209,73 @@ help flag:
     $ inspirehep crawler job --help
 
 
+Operations in QA
+==========================
+
+Migrate records in QA
+-----------------------------
+
+Connect to QA builder machine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. First of all establish a Kerberos authentication (this can be helpfull:
+   http://linux.web.cern.ch/linux/docs/kerberos-access.shtml )
+
+2. After you have run the kinit command and have successfully authenticated you should be able to
+   connect to the builder machine:
+
+.. code-block:: shell
+
+    localhost$ ssh username@inspire-qa-worker3-build1.cern.ch
+
+3. At this point it's a good idea to initialize a screen so you have something to connect to and reestablish your
+   session if something happens to your connection while working remotely to a machine:
+
+.. code-block:: shell
+
+    # This will also reconnect no a running session if any
+    build1$ screen -dR
+
+Remigrate the records
+~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Make sure you have the dump of the records in your local directory(otherwise transfer it there via scp).
+   The accepted format is .xml.gz
+
+2. Now you should workon the virtual enviroment of Inspire:
+
+.. code-block:: shell
+
+    build1$ workon inspire
+
+3. Now you can migrate the records, which will be done using the inspire migrator:
+
+.. note::
+
+      You shouldn't drop the database or destroy the es index as
+      the existing records will be overwritten with the ones introduced.
+
+.. code-block:: shell
+
+    build1$ inspirehep migrator populate -f filename
```

Comment:
https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/migrator/cli.py#L147

Reply:
I should probably take over, it has changed quite a lot since @Glignos did it.